types/stopword/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for stopword 0.3
+// Project: https://github.com/fergiemcdowall/stopword
+// Definitions by: Rico Sandyca Novenza <https://github.com/ricosandyca>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace Stopword {
+    type TStopwords = string[];
```

Comment:
Is the `TStopwords` type strictly necessary? Its actually more characters than `string[]`, so it doesn't seem like it adds much utility.